contentScript.js

```diff
@@ -0,0 +1,6 @@
+var s = document.createElement('script');
+s.src = chrome.extension.getURL('courses.js');
+s.onload = function() {
+    this.remove();
+};
+(document.head || document.documentElement).appendChild(s);
```

Comment:
This is the thread I referenced to make the contentScript.js:
http://stackoverflow.com/a/9517879

More reading (from the same author):
http://stackoverflow.com/a/9636008

This is where he mentions that there is a specific order that certain events fire, which could affect our script changes:
http://stackoverflow.com/a/8994454

I've read elsewhere that this is the case as well, but do not have the threads on hand